src/utils/apiResponse.utils.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export type ControllerResponse = {
+  isSuccess: number;
+  statusCode: number;
+  message: string;
+  data?: any;
+  error?: any;
+};
```

Comment:
PullProofAI

The `ControllerResponse` type definition lacks clarity and consistency in naming conventions.

```diff
- export type ControllerResponse = {
+ export interface ControllerResponse {
```

Using an interface instead of a type for `ControllerResponse` is more aligned with TypeScript best practices. It helps define the shape of objects and provides better extensibility.

### Issue 2


<!-- This is an auto-generated comment by PullProofAI -->